api/management/commands/loaddata.py

```diff
@@ -22,22 +23,66 @@ def handle(self, *args, **options):
             disease_path = os.path.join(options['path'], 'diseases.tsv')
             with open(disease_path) as disease_file:
                 disease_reader = csv.DictReader(disease_file, delimiter='\t')
+                disease_list = []
                 for row in disease_reader:
-                    Disease.objects.create(
+                    disease = Disease(
                         acronym=row['acronym'],
                         name=row['disease']
                     )
+                    disease_list.append(disease)
+                Disease.objects.bulk_create(disease_list)
 
         # Samples
         if Sample.objects.count() == 0:
             sample_path = os.path.join(options['path'], 'samples.tsv')
             with open(sample_path) as sample_file:
                 sample_reader = csv.DictReader(sample_file, delimiter='\t')
+                sample_list = []
                 for row in sample_reader:
                     disease = Disease.objects.get(acronym=row['acronym'])
-                    Sample.objects.create(
+                    sample = Sample(
                         sample_id=row['sample_id'],
                         disease=disease,
                         gender=row['gender'] or None,
                         age_diagnosed=row['age_diagnosed'] or None
                     )
+                    sample_list.append(sample)
+                Sample.objects.bulk_create(sample_list)
+
+        # Genes
+        if Gene.objects.count() == 0:
+            gene_path = os.path.join(options['path'], 'genes.tsv')
+            with open(gene_path) as gene_file:
+                gene_reader = csv.DictReader(gene_file, delimiter='\t')
+                gene_list = []
+                for row in gene_reader:
+                    gene = Gene(
+                        entrez_gene_id=row['entrez_gene_id'],
+                        symbol=row['symbol'],
+                        description=row['description'],
+                        chromosome=row['chromosome'] or None,
+                        gene_type=row['gene_type'],
+                        synonyms=row['synonyms'].split('|') or None,
+                        aliases=row['aliases'].split('|') or None
+                    )
+                    gene_list.append(gene)
+                Gene.objects.bulk_create(gene_list)
+
+        # Mutations
+        if Mutation.objects.count() == 0:
+            mutation_path = os.path.join(options['path'], 'mutation-matrix.tsv.bz2')
+            with bz2.open(mutation_path , 'rt') as mutation_file:
+                mutation_reader = csv.DictReader(mutation_file, delimiter='\t')
+                mutation_list = []
+                for row in mutation_reader:
+                    sample_id = row.pop('sample_id')
+                    sample = Sample.objects.get(sample_id=sample_id)
+                    for entrez_gene_id, mutation_status in row.items():
+                        if mutation_status == '1':
+                            try:
```

Comment:
In the event that this is useful, when I download the [latest genes file](https://raw.githubusercontent.com/cognoma/genes/master/data/genes.tsv), load with `pandas`, and run `4253 in df.entrez_gene_id` I get `True`, but when I run `117153 in df.entrez_gene_id`, I get `False`. I feel as though this further supports the idea that the discrepancy between these two files is due to the dates on which they were made, given that some information from Entrez changed in between.